ReSharper.FSharp/test/src/FSharp.Common.Tests/SciptPsiModulesTest.fs

```diff
@@ -128,3 +129,15 @@ type FSharpFileServiceStub() =
     interface IFSharpFileService with
         member x.IsScratchFile _ = false
         member x.IsScriptLike _ = false
+
+[<SolutionComponent>]
+type TestExtensionTypingProviderStub() =
```

Comment:
Would it be hard to have a single `TestExtensionTypingProviderStub` component? Do I get it right that there's a zoning issue? Ideally we should be trying to remove these stubs instead of adding more. 🙂

Reply:
> Do I get it right that there's a zoning issue?
Exactly

Reply:
What components are missing?